scripts/load_test_customer_data.sh

```diff
@@ -158,7 +155,11 @@ git checkout -- "$NISE_REPO_PATH/examples/ocp_on_azure/ocp_static_data.yml"
 if [[ $USE_OC == 1 ]]; then
     WORKER_POD="${KOKU_WORKER_POD_NAME:-koku-worker-0}"
     oc rsync --delete $KOKU_PATH/testing/pvc_dir/insights_local ${WORKER_POD}:/tmp
-    oc rsync --delete $KOKU_PATH/testing/local_providers/aws_local/* ${WORKER_POD}:/tmp/local_bucket
+    for SOURCEDIR in $(ls -1d $KOKU_PATH/testing/local_providers/aws_local*)
+    do
+        DESTDIR="${WORKER_POD}:$(echo $SOURCEDIR | sed 's#'$KOKU_PATH'#/tmp' | sed 's/aws_local/local_bucket/')"
```

Comment:
Don't you need a slash here?
changing `sed 's#'$KOKU_PATH'#/tmp` to `sed 's/#'$KOKU_PATH'#/tmp`

Reply:
Nope. See: https://www.gnu.org/software/sed/manual/html_node/Regexp-Addresses.html#Regexp-Addresses

```
\%regexp%

    (The % may be replaced by any other single character.)

    This also matches the regular expression regexp, but allows one to use a different delimiter than /. This is particularly useful if the regexp itself contains a lot of slashes, since it avoids the tedious escaping of every /. If regexp itself includes any delimiter characters, each must be escaped by a backslash (\).

    The following commands are equivalent. They print lines which start with ‘/home/alice/documents/’:

    sed -n '/^\/home\/alice\/documents\//p'
    sed -n '\%^/home/alice/documents/%p'
    sed -n '\;^/home/alice/documents/;p'

```

Reply:
Ah. The `/tmp` threw me.